templates/experiments/components/experiment_chat.html

```diff
@@ -42,15 +42,17 @@
                 </td>
               {% endif %}
               {% if perms.annotations.add_usercomment %}
-                <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500"><i class="fa-regular fa-comment"></i> {{message.get_user_comments|length}}</td>
+                <td class="whitespace-nowrap px-3 py-4 text-sm text-gray-500">
```

Comment:
Nicer / more succinct approaches welcome!

Reply:
You could use [hx-swap-oob](https://htmx.org/attributes/hx-swap-oob/) in the 'add_comment' view response to update the comment count.

I used this in pipelines: https://github.com/dimagi/open-chat-studio/blob/ccda7fcc79c3fb103ed098e951be57bffa6fd36a/templates/analysis/components/group_progress_inner.html#L2

Reply:
Of course! Thanks